src/ucp/wireup/select.c

```diff
@@ -1408,13 +1435,19 @@ ucs_status_t ucp_wireup_select_aux_transport(ucp_ep_h ep,
                                              ucp_rsc_index_t *rsc_index_p,
                                              unsigned *addr_index_p)
 {
-    ucp_wireup_criteria_t criteria = {0};
-    double score;
+    ucp_wireup_criteria_t criteria       = {0};
+    ucp_wireup_select_info_t select_info = {0};
+    ucs_status_t status;
 
     ucp_wireup_fill_aux_criteria(&criteria, params);
-    return ucp_wireup_select_transport(ep, address_list, address_count,
-                                       &criteria, -1, -1, -1, -1, 1, rsc_index_p,
-                                       addr_index_p, &score);
+    status = ucp_wireup_select_transport(ep, address_list, address_count,
+                                         &criteria, -1, -1, -1, -1, 1,
+                                         &select_info);
+
+    *rsc_index_p  = select_info.rsc_index;
```

Comment:
maybe ucp_wireup_select_aux_transport should also return ucp_wireup_select_info_t , for consistency?

Reply:
done - moved `ucp_wireup_select_info` structure to wireup.h file